mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -451,6 +439,8 @@ class ALSModel private[ml] (
 @Since("1.6.0")
 object ALSModel extends MLReadable[ALSModel] {
 
+  @transient private[recommendation] val _f2jBLAS = new F2jBLAS
```

Comment:
No more or less than using `ml.linalg.BLAS` - I did think of that but the `var` needs to be exposed as `private[ml]`. If we're ok with that then it'll be slightly cleaner to use that, yes.